### PR TITLE
bogofilter: update to 1.2.5

### PR DESCRIPTION
**Pull request by flyn-org**
Signed-off-by: W. Michael Petullo <mike@flyn.org>

Maintainer: me
Compile tested: x86_64 master bba6646b
Description:

### CODE REVIEW

File: mail/bogofilter/Makefile [hunk 1]
```diff
@@ -8,16 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bogofilter
-PKG_VERSION:=1.2.4
-PKG_RELEASE:=5
+PKG_VERSION:=1.2.5
+PKG_RELEASE:=1
 
 PKG_LICENSE:=GPLv2
```
Comment (neheb) [inline mail/bogofilter/Makefile]: This should be GPL-2.0-or-later

File: mail/bogofilter/Makefile [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
 PKG_CPE_ID:=cpe:/a:bogofilter:bogofilter
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=@SF/bogofilter
-PKG_HASH:=e10287a58d135feaea26880ce7d4b9fa2841fb114a2154bf7da8da98aab0a6b4
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=@SF/project/bogofilter/bogofilter-stable/
+PKG_HASH:=3248a1373bff552c500834adbea4b6caee04224516ae581fb25a4c6a6dee89ea
 
 PKG_INSTALL:=1
```
Comment (neheb) [inline mail/bogofilter/Makefile]: PKG_BUILD_PARALLEL:=1 maybe?

**Comment by flyn-org** [comment]
Updated license field.

**Comment by flyn-org** [comment]
Enabled parallel build.

**Pull request merged**